src/script/components/RichTextEditor/plugins/MentionsPlugin/MentionsPlugin.tsx

```diff
@@ -41,7 +41,7 @@ const TRIGGER = '@';
  * @param text the text in which to look for mentions triggers
  */
 function checkForMentions(text: string): MenuTextMatch | null {
-  const match = new RegExp(`(^| )(${TRIGGER}([\\w ]*))$`).exec(text);
+  const match = new RegExp(`(^| )(${TRIGGER}(\\S*))$`).exec(text);
```

Comment:
This was actually the regexp used before the migration to Lexical (see https://github.com/wireapp/wire-webapp/blob/9684e0593a4b2eec22bea97ccc34701fce3e4219/src/script/util/MentionUtil.ts#L40)